elmo/settings/__init__.py

```diff
@@ -132,6 +132,28 @@
         continue
     globals()[key[len('DJANGO_'):]] = boolmapper.get(value.lower(), value)
 
+# Retrieve pulse.m.o parameters
+for key in (
+    'PULSE_HOST',  # default set in base.py
+    'PULSE_SSL',  # default set in base.py
+    'PULSE_USER',
+    'PULSE_PASSWORD',
+    'PULSE_TTL',
```

Comment:
Is there a reason not to set a default for TTL?

I'm assuming we want no expiration, so if Elmo has an extended outage messages are still there when we come back.

Reply:
I've switched this over to have `None` as default TTL, I think that's a good choice for us.